packages/theme/src/cli/utilities/theme-uploader.ts

```diff
@@ -432,10 +432,8 @@ async function handleFailedUploads(
 function reportFailedUploads(uploadResults: Map<string, Result>) {
   for (const [key, result] of uploadResults.entries()) {
     if (!result.success) {
-      const errorMessage = result.errors?.asset?.map((err) => `-${err}`).join('\n')
-      outputWarn(`Failed to upload file ${key}:`)
-      outputInfo(`${errorMessage}`)
-      outputNewline()
+      const errorMessage = result.errors?.asset?.map((err) => `-${err}`).join('\n') ?? 'File upload failed'
```

Comment:
I think we should remove the dash. It looks odd when used inside of the error block.

```suggestion
      const errorMessage = result.errors?.asset?.map((err) => `${err}`).join('\n') ?? 'File upload failed'
```